### ISSUE TITLE
Automatically skip pom type modules

### ISSUE THREAD

**Issue by adrienlauer** [siom79/japicmp#66]
First I would like to thank you for this great tool.

I plan to use japicmp maven plugin in a full development stack (https://github.com/seedstack) which contains a lot of modules, some of which are of pom packaging type. I configured japicmp in my parent pom with maven properties to automatically check the current version against the latest release:

    <oldVersion>
        <dependency>
            <groupId>${project.groupId}</groupId>
            <artifactId>${project.artifactId}</artifactId>
            <version>RELEASE</version>
        </dependency>
    </oldVersion>
    <newVersion>
        <file>
            <path>${project.build.directory}/${project.build.finalName}.jar</path>
        </file>
    </newVersion>

I found that japicmp doesn't automatically skip pom packaging modules and fails instead because it cannot find the file specified in <newVersion>. I worked around this problem by using a property in the <skip> tag that is set to true by default but I must override it to false in each of my 69 jar modules.

It would help me a lot if the japicmp maven plugin could automatically skip every non jar-packaged module (with an info log). This behavior may be enabled with a configuration option.

Regards.

**Comment by siom79** [comment]
I have pushed a commit to the develop branch that introduces a new option for the maven plugin:
```
<skipPomModules>true</skipPomModules>
```
The default value is `true`, hence you would not have to configure it.

If you like, you can try it out:
```
git clone https://github.com/siom79/japicmp.git
cd japicmp
git checkout develop
mvn clean install
```
Let me know what you think.

**Comment by adrienlauer** [comment]
I'm happy to say that the new option is working as intended. I tested it in a branch of the main framework with a lot of modules (a mix of pom and jar types) and everything went smooth.

Many thanks.

**Comment by siom79** [comment]
Released with version 0.5.2.
